docs/getting-started/roadmap.md

```diff
@@ -1,20 +1,31 @@
 # Roadmap
 
-## Release 1.0 onwards
+This document defines a high level roadmap for KubeEdge development.
 
-KubeEdge will provide the fundamental infrastructure and basic functionality for IOT/Edge workloads. This includes: 
+The [milestones defined in GitHub](https://github.com/kubeedge/kubeedge/milestones) represent the most up-to-date plans.
 
-- Istio-based service mesh across Edge and Cloud where micro-services can communicate freely in the mesh.
+KubeEdge 1.0 is our current stable branch. The roadmap below outlines new features that will be added to KubeEdge.
+
+## Release 1.1
+- Replace data exchange format between cloud and edge from json to protobuf.
+- Support reliable message delivery from cloud to edge.
+- Evaluate gRPC for cloud to edge communication.
+- Support CSI for persistent storage (using PV/PVC/StorageClass) at edge.
+- Support ingress at edge.
+- Add admission-webhook based validation for device CRDs.
 - Enhance performance and reliability of KubeEdge infrastructure.
+- Upgrade Kubernetes dependencies in vendor to v1.15.
+- Migrate to Go module for dependency management.
+- Improve contributor experience by defining project governance policies, release process, membership rules etc.
+- Improve the performance and e2e tests with more metrics and scenarios.
+
+## Future
```

Comment:
add the monitoring and telemetry for edge here

Reply:
updated.

Reply:
@fisherxu , updated, thanks.